api/handler/response.go

```diff
@@ -34,7 +34,7 @@ type ListObjectsV2Response struct {
 	XMLName               xml.Name       `xml:"http://s3.amazonaws.com/doc/2006-03-01/ ListBucketResult" json:"-"`
 	CommonPrefixes        []CommonPrefix `xml:"CommonPrefixes"`
 	Contents              []Object       `xml:"Contents"`
-	ContinuationToken     string         `xml:"ContinuationToken,omitempty"`
+	ContinuationToken     string         `xml:"ContinuationToken"`
```

Comment:
Why we need this change? If I understand correctly `ContinuationToken` is included in response only if it was in request.

Reply:
Yep, but how we can differ when ContinuationToken is set as empty and when ContinuationToken is not set? 
It has not passed  s3tests_boto3.functional.test_s3:test_bucket_listv2_continuationtoken_empty because of it. 
I'm not sure it matters a lot if we return a response with empty continuation token.

Reply:
Then I'm not sure if we need to pass such test. @alexvanin ?

Reply:
Let's see what actual AWS will tell us about it (looking into it).

Reply:
Minio response with an error
[kira@kira-20n4000brt /home/kira/work/neofs-s3-gw]$ aws s3api list-objects-v2 --bucket start --endpoint-ur http://127.0.0.1:9000 --continuation-token ""

An error occurred (InvalidArgument) when calling the ListObjectsV2 operation: The continuation token provided is incorrect
[kira@kira-20n4000brt /home/kira/work/neofs-s3-gw]$ aws s3api list-objects-v2 --bucket start --endpoint-ur http://127.0.0.1:9000 --continuation-token ''

An error occurred (InvalidArgument) when calling the ListObjectsV2 operation: The continuation token provided is incorrect

Reply:
AWS eu-north-1 returns the same error.

```
$ aws --endpoint-url https://s3.eu-north-1.amazonaws.com s3api list-objects-v2  --bucket avbexamplebucket --continuation-token ''

An error occurred (InvalidArgument) when calling the ListObjectsV2 operation: The continuation token provided is incorrect
```

I suggest to keep `omitempty` in this PR and make a note about this test in the docs. 
In separate issue we can mimic this behavior with error output.